tests/ci_folder_parsing.py

```diff
@@ -449,6 +449,228 @@ def test_empty(self, steps, local_class, operating_system, token=None):
         "ShowSdwanSystemStatus": True,
         "ShowSdwanSoftware": True,
     },
+    "nxos": {
+        "aci": {
+            "ShowPlatformInternalHalPolicyRedirdst": True,
+            "ShowServiceRedirInfoGroup": True,
+        },
+        "ShowMacAddressTableBase": True, # missing test
```

Comment:
they are not migrated yet. and will be coming on new PR, correct?

Reply:
True for all but the ones that are missing tests already (e.g. I am not looking to build tests that do not exist yet)

Reply:
thanks for the reply. yes, you don't need to build ones which don't have even old style unittest.